### DISCUSSION TITLE
Hello! Can you provide a version of chromeos.

### DISCUSSION THREAD

**Discussion by GordonCHN** [rejetto/hfs#39] [Ideas]
People who use chromeos need this too much to be a file sharing tool.
hfs for Linux runing chromeos crosh show "bash: ./hfs: Permission denied"

**Comment by rejetto** [comment]
I don't have a chromeos myself.
It would be nice if a chromeos user could contribute a solution to this.

In the meantime, I searched and found this comand that should get you node.js on your computer
`curl -fsSL https://rpm.nodesource.com/setup_16.x | sudo bash -`

If this works, then you should be able to run the hfs-node.zip distribution, by running the `run` file..
You may need to `chmod +x run` first time.

**Comment by redtrillix** [comment]
I was under the assumption that you could execute Linux packages on ChromeOS.
